lms/static/sass/shared/_footer-edx.scss

```diff
@@ -392,24 +595,96 @@ footer#footer-edx-v3 {
       @include margin-left($baseline*0.25);
     }
 
-    .social {
-      display: inline;
-      margin-top: 0;
+    .footer-language-selector {
+      display: inline-block;
+      float: none;
     }
+  }
 
+  @include media-breakpoint-up(md) {
+    .ml-auto {
+      margin-left: auto !important;
+    }
 
     .footer-language-selector {
       display: inline-block;
-      float: none;
+      width: max-content;
     }
 
-    ul.social li {
-      display: inline;
-      width: 0;
+    .column-1 {
+      flex: 0 0 25%;
+      max-width: 25%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    .column-2 {
+      flex: 0 0 75%;
+      max-width: 75%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    .column-2-2 {
+      flex: 0 0 80%;
+      max-width: 80%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+
+      @include margin-left(0);
+      //flex-basis: auto;
+    }
+
+    .column-3 {
+      flex: 0 0 100%;
+      max-width: 100%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    .column-2-1 {
+      flex: 0 0 33.3333%;
+      max-width: 33.3333%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+      //flex-basis: auto;
+    }
+
+    .column-3-1 {
+      flex: 0 0 75%;
+      max-width: 75%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    .column-3-2 {
+      flex: 0 0 100%;
+      max-width: 100%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
```

Comment:
Same comment re. updating to be DRY.

Reply:
Most of these styles are the defaults for the column and haven't changed so there is no need to declare again.